lib/mkDummySrc.nix

```
@@ -122,6 +122,13 @@ let
 
   dummyrs = writeText "dummy.rs" ''
     #![allow(dead_code)]
+    #![cfg_attr(target_os = "none", no_std)]
+
+    #[cfg_attr(target_os = "none", panic_handler)]
+    fn panic(_info: &core::panic::PanicInfo<'_>) -> ! {
```

Probably a good idea to make the namespace explicit.

```suggestion
    fn panic(_info: &::core::panic::PanicInfo<'_>) -> ! {
```